_posts/2020-11-07-for-loops-how-to.md

```diff
@@ -0,0 +1,56 @@
+---
+layout: post
+title:  "How to be successful with for loops"
+date:   2020-11-07 12:54:04 -0700
+categories: jekyll update
+---
+In this post you will learn about the basics of creating for loops and why they
+are important as a bioinformatics tool.
```

Comment:
In your conclusion I would add a sentence or two to highlight why it can be useful in a bioinformatics pipeline.